### ISSUE TITLE
".csproj file does not exist" when starting bot 

### ISSUE THREAD

**Issue by Primordial-47** [microsoft/BotFramework-Composer#9070]
.csproj file does not exist when running Local Runtime in Bot framework composer.

"Error occurred building the bot

Message:
Command failed: dotnet user-secrets init --project NextBot.csproj
Command failed : The project file 'C:\Users\kwilkins-admin\BotComposer\NextBot\NextBot.csproj' does not exist.
Command:
dotnet user-secrets init --project NextBot.csproj
Stack Trace:
Error: Command failed: dotnet user-secrets init --project NextBot.csproj
Command failed : The project file 'C:\Users\kwilkins-admin\BotComposer\NextBot\NextBot.csproj' does not exist.

    at ChildProcess.exithandler (child_process.js:308:12)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
Output:
System.IO.FileNotFoundException: The project file 'C:\Users\kwilkins-admin\BotComposer\NextBot\NextBot.csproj' does not exist.
   at Microsoft.Extensions.SecretManager.Tools.Internal.MsBuildProjectFinder.FindMsBuildProject(String project)
   at Microsoft.Extensions.SecretManager.Tools.Internal.InitCommand.ResolveProjectPath(String name, String path)
   at Microsoft.Extensions.SecretManager.Tools.Internal.InitCommand.Execute(CommandContext context)
   at Microsoft.Extensions.SecretManager.Tools.Internal.InitCommand.Execute(CommandContext context, String workingDirectory)
   at Microsoft.Extensions.SecretManager.Tools.Internal.InitCommandFactory.Execute(CommandContext context, String workingDirectory)
   at Microsoft.Extensions.SecretManager.Tools.Program.RunInternal(String[] args)
   at Microsoft.Extensions.SecretManager.Tools.Program.TryRun(String[] args, Int32& returnCode)
Killed:
Code:
1
Stderr:
Command failed : The project file 'C:\Users\kwilkins-admin\BotComposer\NextBot\NextBot.csproj' does not exist."

## Version

Version: 2.2-nightly.295743.7333498
NodeJS: 12.13.0
V*: 8.0.426.27-electron.0
## Browser

- [x ] Electron distribution: Electron: 8.2.4
- [x ] Chrome: Chrome: 80.0.3987.165

## OS

- [ ] macOS
- [x ] Windows
- [ ] Ubuntu

## To Reproduce

1. Go to 'start bot'
2. Click on 'start bot'
3. See error

## Expected behavior

![image](https://user-images.githubusercontent.com/102365604/160125465-68fe3b64-99d8-4234-bde1-d6a1f8e2aba2.png)

## Screenshots

![image](https://user-images.githubusercontent.com/102365604/160125647-c7dc1188-cbb4-4c0d-8ce5-446d64b42f69.png)

## Additional context

Works when I start a new bot, but using an shared bot from a colleague.

**Comment by dmvtech** [comment]
@Primordial-47 

Have you checked to see if the file is actually there or missing? On the file system. This screams to me that this is a **_source control_** type of issue. Like `.csproj` files are in a `.gitignore` file or something and isn't getting checked in/commited.

**Comment by Primordial-47** [comment]
> @Primordial-47
> 
> Have you checked to see if the file is actually there or missing? On the file system. This screams to me that this is a **_source control_** type of issue. Like `.csproj` files are in a `.gitignore` file or something and isn't getting checked in/commited/tracked.

Thanks for reaching out Dana. I did check and confirm yesterday that the file is locatable. 

However, I think the issue may have been in the extraction process (or source control as you mentioned). Initially I extracted the contents to a folder and then dragged them to a subfolder. I just deleted that folder, created a new one and pointed the extraction to it. Loaded BFC up and started local runtime and it worked. Thanks again for your response!
